gui/mozregui/log_report.py

```diff
@@ -9,7 +16,17 @@ def on_log_received(self, data):
         time_info = datetime.fromtimestamp((data['time']/1000)).isoformat()
         log_message = '%s: %s : %s' % (
             time_info, data['level'], data['message'])
-        self.appendPlainText(log_message)
+        fmt = QTextCharFormat()
+        fmt.setForeground(color[data['level']])
```

Comment:
hmm, according to the doc (http://mozbase.readthedocs.org/en/latest/mozlog_structured.html#data-format), it would be possible to have log levels like CRITICAL, ERROR, WARNING, INFO, DEBUG. So this would crash if we (or one of mozregression dependency) use CRITICAL or ERROR as it is not handled. You could set the same color as for WARNING though.

Also, I think it is possible for somebody using mozlog to add its own level. so we should try to get the color from the COLORS dict - and only apply the color if we find one for the given level.